server.js

```diff
@@ -0,0 +1,6 @@
+const chalk = require('chalk');
```

Comment:
It looks like you're merging in a lot of already existing changes. Can you update this branch with the latest master?